common/djangoapps/track/views/__init__.py

```diff
@@ -76,6 +79,7 @@ def user_track(request):
     context_override['page'] = page
 
     with eventtracker.get_tracker().context('edx.course.browser', context_override):
+        log.info(u"Event emitted '%s' for %s at page: %s ", name, username, page)
```

Comment:
This will just duplicate the tracking log events in the system log. That doesn't seem to help us figure out why it's happening.

I think we need to instrument the client-side js to try to detect when "illegal" event sequences are happening, and log more info about the state of the video player.
 Illegal sequences are things like "play_video" -> "play_video" (once you're playing, shouldn't see another play without a pause or stop in between). I'm not sure what player state would be helpful to log. We could at least log something like "SAW_WEIRD_STATE", so we could easily find them. If it logged to the browser console in some sort of debug mode, we might have a better chance of reproducing the issue locally.

Do you or @adampalay know how the js side of the player works? Any thoughts for what to log to figure out why we're getting weird sequences of events?